aws/aws-how-to/instances/find-ubuntu-images.rst

```diff
@@ -35,7 +35,7 @@ Images for EC2 and EKS
          ubuntu/$PRODUCT/$RELEASE/stable/current/$ARCH/$VIRT_TYPE/$VOL_TYPE/ami-id
 
       * PRODUCT: `server`, `server-minimal`, `pro-server` or `pro-minimal`
-      * RELEASE: `jammy`, `22.04`, `focal`, `20.04`, `bionic`, `18.04`, `xenial`, or `16.04`
+      * RELEASE: `noble`, 24.04, `jammy`, `22.04`, `focal`, `20.04`
```

Comment:
xenial and bionic are still in ESM. i don't think we should drop these.

Reply:
I am not totally convinced to drop them for the same reasons, but I believe we should push the users a little to get newer versions. ESM customers are already running workloads. Also, it is easy to get the pattern, so my suggestion is to keep the latest three or current LTSs only.

Reply:
does the ordering of suites already suggest a recommended image to use? i don't want to encourage people to use older suites, but also don't want to suggest that these suites aren't supported for people who do need them